internal/input/ledger_range.go

```diff
@@ -0,0 +1,159 @@
+package input
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/stellar/go/exp/ingest/ledgerbackend"
+	"github.com/stellar/stellar-etl/internal/utils"
+)
+
+type graphPoint struct {
```

Comment:
It'll be helpful to add a comment explaining this struct (though un-exported), given how heavily it's used below. This will help guide future maintenance and explain initial design decisions.